tests/queries/0_stateless/00515_enhanced_time_zones.reference

```diff
@@ -1,6 +1,6 @@
-2017-12-05 10:37:47
-2017-12-05 10:37:47
-2017-12-05 10:37:47
+2017-12-05 10:37:47+05:30
+2017-12-05 10:37:47+05:30
+2017-12-05 10:37:47+05:30
```

Comment:
can we parse this time format correctly from a string ?

Reply:
According to the test case, we can do this
![image](https://github.com/user-attachments/assets/808c7e19-1a75-4eb3-803e-977b0ff01fb7)